configurations/luminate_pipeline.py

```diff
@@ -0,0 +1,257 @@
+from core_data_modules.cleaners import swahili
+from src.pipeline_configuration_spec import *
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="Luminate",
+    test_participant_uuids=[
+        "avf-participant-uuid-ced0f78f-f989-42e4-8813-b1ead4fff0ae",
+        "avf-participant-uuid-d38dcb19-5170-4441-99de-5aa3e9bcafc1",
+        "avf-participant-uuid-2fa53f8c-8f71-490c-8aff-40c6929b2675", 
+        "avf-participant-uuid-7d817591-37b9-43ef-b3c3-303fdfa1544f", 
+        "avf-participant-uuid-f24811c3-90a6-4fcd-bf32-e8b38eb98ca4", 
+        "avf-participant-uuid-9fe96ca0-18ba-474c-b86f-c4709f45d4ca", 
+        "avf-participant-uuid-dbc84ba4-f55f-4741-b1c8-7b87727f19e1", 
+        "avf-participant-uuid-b972d5f2-be30-4eea-be1c-a4d973a15330", 
+        "avf-participant-uuid-73b4f00a-5e49-418d-896f-95185f59fe4d", 
+        "avf-participant-uuid-7d144e9e-b54f-4d1f-bca9-3e8cdeeaedcc", 
+        "avf-participant-uuid-88ef05ba-4c56-41f8-a00c-29104abab73e", 
+        "avf-participant-uuid-5ca68e07-3dba-484b-a29c-7a6c989036b7"
+    ],
+    engagement_database=EngagementDatabaseClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-engagement-databases-firebase-credentials-file.json",
+        database_path="engagement_databases/POOL-KENYA"
+    ),
+    uuid_table=UUIDTableClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+        table_name="avf-global-urn-to-participant-uuid",
+        uuid_prefix="avf-participant-uuid-"
+    ),
+    operations_dashboard=OperationsDashboardConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-dashboards-firebase-adminsdk-gvecb-ef772e79b6.json",
+    ),
+    rapid_pro_sources=[
+        RapidProSource(
+            rapid_pro=RapidProClientConfiguration(
+                domain="textit.com",
+                token_file_url="gs://avf-credentials/pool-kenya-2-textit-token.txt"
+            ),
+            sync_config=RapidProToEngagementDBConfiguration(
+                flow_result_configurations=[
+                    FlowResultConfiguration("Luminate_s01_demogs", "pool_kenya_location", "location"),
+                    FlowResultConfiguration("Luminate_s01_demogs", "pool_kenya_gender", "gender"),
+                    FlowResultConfiguration("Luminate_s01_demogs", "pool_kenya_age", "age"),
+                    FlowResultConfiguration("Luminate_s01_demogs", "pool_kenya_disabled", "disabled"),
+                    FlowResultConfiguration("Luminate_s01_demogs", "preferred_language", "preferred_language"),
+
+                    FlowResultConfiguration("Luminate_s01e01_activation", "rqa_s01e01", "luminate_s01e01")
+                ],
+            )
+        )
+    ],
+    google_form_sources=[],
+    kobotoolbox_sources=[],
```

Comment:
Consider deleting these lines for simplicity

Reply:
updated, thanks.